app/queues/reconciler/reconciler-queue.server.ts

```diff
@@ -0,0 +1,43 @@
+import logger from '~/lib/logger.server';
+import { reconcilerQueue } from './reconciler-worker.server';
+
+declare global {
+  var __reconciler_queue_init__: boolean;
+}
+
+reconcilerQueue.on('error', (err) => {
+  logger.error('Reconciler encountered an error', err);
+});
+
+// function to add jobs
+export const addReconcilerJob = async () => {
+  if (process.env.NODE_ENV !== 'production' && global.__reconciler_queue_init__) {
+    // Only do this setup once if in dev
```

Comment:
Out of curiosity, does this actually run multiple times in production? Why?

Reply:
Nope, on prod it should only run once every startup
If you check that condition, (an early return) it only returns if not priduction and already initialized

Reply:
Oh, so it could run multiple times in dev, so this prevents that. I suppose that happens due to live reload?